server/websocket.go

```diff
@@ -547,6 +549,13 @@ func (p *Plugin) handleJoin(userID, connID, channelID, title string) error {
 		}, &model.WebsocketBroadcast{ChannelId: channelID, ReliableClusterSend: true})
 	}
 
+	if userID == p.getBotID() && state.Call.Recording != nil {
+		p.publishWebSocketEvent(wsEventCallRecordingStart, map[string]interface{}{
+			"callID":  channelID,
+			"startAt": state.Call.Recording.StartAt,
+		}, &model.WebsocketBroadcast{ChannelId: channelID, ReliableClusterSend: true})
+	}
```

Comment:
I suppose this could cause interesting states, if we send this message but the recording actually fails to start. Would it be better to send this only after the recording system has successfully started? (It would let us also send "failed" messages (with error/debug info too?), and hosts could try to start the recording again)